wazo_call_logd/plugins/cdr/schemas.py

```diff
@@ -61,6 +61,15 @@ class RecordingMediaExportSchema(Schema):
     uuid = fields.UUID()
 
 
+class DestinationDetailsSchema(Schema):
+    type = fields.String(required=True, default='unknown')
+    user_uuid = fields.UUID()
+    user_name = fields.String()
+    meeting_uuid = fields.UUID()
+    meeting_name = fields.String()
+    conference_id = fields.Integer()
```

Comment:
Can you try to make one schema per kind of destination?
It will avoid confusion for future reader
ex: 
* `DestinationUserDetails`
* `DestinationMeetingDetails`
* etc..
Same thing of function keys

Reply:
Oh yes, +1!!! Is it clear for you Aniss?

Reply:
like this right?

```python

class DestinationUserDetails(Schema):
    type = fields.String(required=True, default='unknown')
    user_uuid = fields.UUID()
    user_name = fields.String()

class DestinationMeetingDetails(Schema):
    type = fields.String(required=True, default='unknown')
    meeting_uuid = fields.UUID()
    meeting_name = fields.String()

class DestinationConferenceDetails(Schema):
    type = fields.String(required=True, default='unknown')
    conference_id = fields.Integer()
```

and then here I am not sure how to nest them??, any suggestion :) :

```
destination_details = fields.Nested(DestinationDetailsSchema)
```

Reply:
Yes, that's the idea. You can see an example here: https://github.com/wazo-platform/wazo-confd/blob/master/wazo_confd/plugins/func_key/schema.py#L224-L259

You can also share the `type = fields.String(...)` code using a base class.

Reply:
Actually I did it like the following; in my opinion it is simpler :) 

```python

class DestinationDetailsSchema(Schema):
    type = fields.String(required=True, default='unknown')


class DestinationUserDetails(DestinationDetailsSchema):
    user_uuid = fields.UUID()
    user_name = fields.String()


class DestinationMeetingDetails(DestinationDetailsSchema):
    meeting_uuid = fields.UUID()
    meeting_name = fields.String()


class DestinationConferenceDetails(DestinationDetailsSchema):
    conference_id = fields.Integer()

```

Reply:
Well, it is simpler, but it is also dead code: the classes are not used anywhere... We must use those new classes in the code. The point is to be able to add a new schema class easily without having to change anything else in the schemas.